services/frontend-v3/src/components/privacyModal/PrivacyModalView.jsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Modal } from "antd";
+import { LockOutlined, CheckOutlined, CloseOutlined } from "@ant-design/icons";
+import { FormattedMessage } from "react-intl";
+import ChangeLanguage from "../changeLanguage/ChangeLanguage";
+import "./PrivacyModalView.scss";
+
+const PrivacyModalView = ({ handleOk, handleCancel, showModal, locale }) => {
+  return (
+    <Modal
+      title={
+        <>
+          <LockOutlined /> <FormattedMessage id="privacy.modal.header" />
+          <div className="privacyModalHeaderExtra">
+            <ChangeLanguage />
+          </div>
+        </>
+      }
+      visible={showModal}
+      closable={false}
+      maskClosable={false}
+      okText={
+        <>
+          <CheckOutlined className="ModalbBtnIcon" />
+          <FormattedMessage id="privacy.modal.accept" />
+        </>
+      }
+      cancelText={
+        <>
+          <CloseOutlined className="ModalbBtnIcon" />
+          <FormattedMessage id="privacy.modal.decline" />
+        </>
+      }
+      onOk={handleOk}
+      onCancel={handleCancel}
+      width={700}
+    >
+      <div className="privacyModalContent">
+        {locale === "ENGLISH" ? (
+          <>
+            <p>
+              Please be advised that by logging in you have the ability to
+              self-administer information about yourself. The collection of your
+              self-administered information is known as your Profile information
+              to showcase your professional history. You will be able to control
```

Comment:
Just wondering if we should move this to the i18. This way if we ever need to make an update or add a language it would be easier and keeps the code clean

Reply:
the reason I left it in here for now is that I'm waiting for the final version of the "terms" to see the formatting and how its broken up. once it is finalized i will see how to put it in the translations file. this is essentially a place holder